cli.js

```diff
@@ -356,23 +356,23 @@ else if (command.match(/^local$/i) && argv[1]) {
 // GLOBAL globally use the specified node version
 else if (command.match(/^global$/i) && argv[1] || argv[0] && !argv[1]) {
   spec = argv[1] || argv[0];
-  n.setGlobal(spec, function(err) {
-    if(err) abort(err.message+'. Sorry.');
-    console.log(spec);
-    n.resolveVersionLocally(spec, function(er, found) {
-      if(found) {
-        exit();
-      }
-      n.resolveVersion(spec, function(er, version) {
-        console.log("Installing "+version)
-        n.install(version, function(er) {
-          if(er) return abort(er.message+'. Sorry.')
+  if(err) abort(err.message+'. Sorry.');
+  console.log(spec);
+  n.resolveVersionLocally(spec, function(er, found) {
+    if(found) {
+      exit();
```

Comment:
This will bail early, if the version is installed already. As it is, it won't set the global version requirement in this cae, though.